frontend/components/Setup/SetupWelcome.tsx

```diff
@@ -114,6 +114,7 @@ export const SetupWelcomeLogin = () => {
     async ({ password }: { password: string }) => {
       setIsLoggingIn(true);
       AccountService.loginAccount(password)
+        .then(() => updateWallets())
```

Comment:
this will be resolved here another way https://github.com/valory-xyz/olas-operate-app/pull/153
so maybe you can remove this